ecs-init/exec/iptables/iptables.go

```diff
@@ -77,12 +80,19 @@ func (route *NetfilterRoute) Create() error {
 	}
 
 	if !skipLocalhostTrafficFilter() {
-		err = route.modifyNetfilterEntry(iptablesTableFilter, iptablesInsert, getInputChainArgs)
+		err = route.modifyNetfilterEntry(iptablesTableFilter, iptablesInsert, getLocalhostTrafficFilterInputChainArgs)
 		if err != nil {
 			return err
 		}
 	}
 
+	if !allowOffhostIntrospection() {
+		err = route.modifyNetfilterEntry(iptablesTableFilter, iptablesInsert, getBlockIntrospectionOffhostAccessInputChainArgs)
+		if err != nil {
+			log.Errorf("Error adding input chain entry to block offhost introspection access: %v", err)
```

Comment:
Should we return the error here? 
Just logging it means to let the previous behavior go, right?

Reply:
my intention is to make this "best effort" so only log the error and continue here instead of failing. one situation i can think of is that customer can rename "eth0" to something else. in that case it doesn't seem necessary for us to block them